client/src/CodeInput.js

```diff
@@ -10,11 +15,16 @@ const CodeInput = () => {
                 <LanguageSelection />
                 <FileUpload />
             </div>
-            <div className="content">
-                <div className="ui input">
-                    <input type="text" placeholder="Some code..." />
-                </div>
-            </div>
+            <CodeMirror
+                options={{
+                    mode: 'clike',
```

Comment:
This is cool! I'm guessing this is where you can configure what type of syntax highlighting you want to do?